HW_3/forms.py

```diff
@@ -0,0 +1,13 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, BooleanField, SubmitField, DateField
+from wtforms.validators import DataRequired, Email, Length, EqualTo, Regexp
+
+
+class RegistrationForm(FlaskForm):
+    name = StringField('Name', validators=[DataRequired()])
+    surname = StringField('Surname', validators=[DataRequired()])
+    email = StringField('E-mail', validators=[DataRequired(), Email()])
```

Comment:
Тут лучше использовать `EmailField`, хоть и полноценная валидация у вас будет проходить иначе. Это не повредит
```suggestion
    email = EmailField('E-mail', validators=[DataRequired(), Email()])
```